app/assets/javascripts/app/views/notifications_view.js

```diff
@@ -52,7 +52,7 @@ app.views.Notifications = Backbone.View.extend({
     var change = unread ? 1 : -1,
         all_notes = $('ul.nav > li:eq(0) .badge'),
         type_notes = $('ul.nav > li[data-type=' + type + '] .badge'),
-        header_badge = $('#notification_badge .badge_count'),
+        header_badge = $('#notification-badge .badge_count'),
```

Comment:
Identifier 'header_badge' is not in camel case.<br>Mixed double and single quotes.